client/src/components/Logout.js

```diff
@@ -0,0 +1,21 @@
+//import React from "react";
+import React, { Component } from "react";
+//import { Nav, Navbar, NavItem } from "react-bootstrap";
+//import { LinkContainer } from "react-router-bootstrap";
+//import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+
+class Logout extends Component {
```

Comment:
This component is probably not needed anymore, so this file can be removed as well.